src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -667,31 +667,64 @@ private static bool IsFromFormAttributeUsedWithIFormFile(ApiParameterDescription
 
         private static readonly IReadOnlyCollection<KeyValuePair<string, string>> ResponseDescriptionMap = new[]
         {
-           new KeyValuePair<string, string>("1\\d{2}", "Information"),
+            new KeyValuePair<string, string>("100", "Continue"),
+            new KeyValuePair<string, string>("101", "Switching Protocols"),
+            new KeyValuePair<string, string>("1\\d{2}", "Information"),
 
+            new KeyValuePair<string, string>("200", "OK"),
             new KeyValuePair<string, string>("201", "Created"),
             new KeyValuePair<string, string>("202", "Accepted"),
+            new KeyValuePair<string, string>("203", "Non-Authoritative Information"),
             new KeyValuePair<string, string>("204", "No Content"),
+            new KeyValuePair<string, string>("205", "Reset Content"),
+            new KeyValuePair<string, string>("206", "Partial Content"),
             new KeyValuePair<string, string>("2\\d{2}", "Success"),
 
+            new KeyValuePair<string, string>("300", "Multiple Choices"),
+            new KeyValuePair<string, string>("301", "Moved Permanently"),
+            new KeyValuePair<string, string>("302", "Found"),
+            new KeyValuePair<string, string>("303", "See Other"),
             new KeyValuePair<string, string>("304", "Not Modified"),
+            new KeyValuePair<string, string>("305", "Use Proxy"),
+            new KeyValuePair<string, string>("307", "Temporary Redirect"),
+            new KeyValuePair<string, string>("308", "Permanent Redirect"),
             new KeyValuePair<string, string>("3\\d{2}", "Redirect"),
 
             new KeyValuePair<string, string>("400", "Bad Request"),
             new KeyValuePair<string, string>("401", "Unauthorized"),
+            new KeyValuePair<string, string>("402", "Payment Required"),
             new KeyValuePair<string, string>("403", "Forbidden"),
             new KeyValuePair<string, string>("404", "Not Found"),
             new KeyValuePair<string, string>("405", "Method Not Allowed"),
             new KeyValuePair<string, string>("406", "Not Acceptable"),
+            new KeyValuePair<string, string>("407", "Proxy Authentication Required"),
             new KeyValuePair<string, string>("408", "Request Timeout"),
             new KeyValuePair<string, string>("409", "Conflict"),
+            new KeyValuePair<string, string>("410", "Gone"),
+            new KeyValuePair<string, string>("411", "Length Required"),
+            new KeyValuePair<string, string>("412", "Precondition Failed"),
+            new KeyValuePair<string, string>("413", "Payload Too Large"),
```

Comment:
```suggestion
            new KeyValuePair<string, string>("413", "Content Too Large"),
```

Reply:
Thanks, I missed that in RFC9110. Should we do the same for 422 Unprocessable Content? (I've added it as Unprocessable Entity.)

Reply:
Good spot - I missed that one.

Reply:
Fixed that too.